### PR TITLE
docs(scanv4): Pretty error message for missing CA

### PR DESCRIPTION
**Pull request by mclasmeier**
## Description

This PR makes sure that an informative error message is displayed to the user in case the CA is missing during `helm upgrade`.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### Here I tell how I validated my change

TODO(replace-me)  
Use this space to explain **how you validated** that **your change functions exactly how you expect it**.
Feel free to attach JSON snippets, curl commands, screenshots, etc. Apply a simple benchmark: would the information you
provided convince any reviewer or any external reader that you did enough to validate your change.

It is acceptable to assume trust and keep this section light, e.g. as a bullet-point list.

It is acceptable to skip testing in cases when CI is sufficient, or it's a markdown or code comment change only.  
It is also acceptable to skip testing for changes that are too taxing to test before merging. In such case you are
responsible for the change after it gets merged which includes reverting, fixing, etc. Make sure you validate the change
ASAP after it gets merged or explain in PR when the validation will be performed.  
Explain here why you skipped testing in case you did so.

Have you created automated tests for your change? Explain here which validation activities you did manually and why so.

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: image/templates/helm/shared/templates/_crypto.tpl [hunk 1]
```diff
@@ -87,7 +87,17 @@
       {{ $_ = set $result "Key" $key }}
     {{ else }}
       {{ if not $._rox._ca }}
-        {{ include "srox.fail" (printf "Tried to generate certificate for %s, but no CA certificate is available." $spec.CN) }}
+        {{ $msg := list
+            (printf "Tried to generate certificate for %s, but no CA certificate is available." $spec.CN)
+            ""
+            "Providing the CA is required when certificates for new components need to be issued. This can happen, for instance, when enabling a new component such as Scanner V4 using 'helm upgrade'."
+            ""
+            "If the CA was generated by the Helm chart during initial installation, then you need to retrieve these auto-generated values, including the CA, from the cluster and provide them to the 'helm upgrade' command. A command for retrieving the auto-generated values was included in the post-installation notes of the central-services Helm chart."
```
[the remark this answers is not in the document]
Comment (mclasmeier) [reply image/templates/helm/shared/templates/_crypto.tpl]: I mean, it cannot reproduced reliably. With a huge probability, yes. But I would say the post installation notes are pretty clear on the nature of these generated values..

**Review by johannes94 — APPROVED** [review]

**Comment by mclasmeier** [comment]
FYI @kcarmichael08. Might be useful for the work on the documentation?

See https://github.com/stackrox/stackrox/pull/10131/files.
I have replaced a very short and generic error message with a more informative one.

**Comment by kcarmichael08** [comment]
Don't hate me, but I noticed that "achieved" was misspelled:

            "If the CA was generated outside of the Helm chart and provided during the installation of the central-services chart, then the same must be done when executing 'helm upgrade'. This can be achived conveniently by specifying the '--reuse-values' flag to 'helm upgrade'."

**Pull request merged**